### PR TITLE
MIS MRC adjacencies for diagonal D1/E1 ramps; INRUL fix for D2 wide ramp

### PR DESCRIPTION
**Pull request by McDuell**
added stability code for MIS MRC adjacent to diagonal D1/E1 ramp branch.

image

image

### CODE REVIEW

**Comment by jflann** [comment]
I suspect that Sec7i_FLEXRamp is not the best place for the MRC adjacency code.  Take a look at some of the other files in there with 'ramp' in the name, I think that's where adjacencies should be handled.  @TarkusSC4 could confirm

**Comment by McDuell** [comment]
The reason I put the code in the Sec7i_FLEXRamp is because there was already existing adjacency code for the A1 that wasn't done by me. Thus I thought that this file would be right place to also contain the D1/E1 adjacencies.

**Comment by jflann** [comment]
In that case, seems good to me.

File: Controller/RUL2/07_RHW/Sec7i_FLEXRamp/Sec7i_FLEXRamp.txt [hunk 1]
```diff
@@ -23702,7 +23702,277 @@
...
+;RHW-8c
+0x57899500,3,0,0x57905400,2,1=0x57899500,3,0,0x57925400,2,1
+0x57899500,3,0,0x57905400,0,0=0x57899500,3,0,0x57925480,0,0
+0x57899580,1,1,0x57905400,0,0=0x57899580,1,1,0x57925400,0,0
+0x57899580,1,1,0x57905400,2,1=0x57899580,1,1,0x57925480,2,1
+;RHW-10c
+0x578a9500,3,0,0x57905400,2,1=0x5789a500,3,0,0x57925400,2,1
+0x578a9500,3,0,0x57905400,0,0=0x5789a500,3,0,0x57925480,0,0
+0x578a9580,1,1,0x57905400,0,0=0x5789a580,1,1,0x57925400,0,0
+0x578a9580,1,1,0x57905400,2,1=0x5789a580,1,1,0x57925480,2,1
```
Comment (jflann) [inline Controller/RUL2/07_RHW/Sec7i_FLEXRamp/Sec7i_FLEXRamp.txt:23812]: Transposition error @McDuell 
```
0x578a9500,3,0,0x57905400,2,1=0x578a9500,3,0,0x57925400,2,1
0x578a9500,3,0,0x57905400,0,0=0x578a9500,3,0,0x57925480,0,0
0x578a9580,1,1,0x57905400,0,0=0x578a9580,1,1,0x57925400,0,0
0x578a9580,1,1,0x57905400,2,1=0x578a9580,1,1,0x57925480,2,1
```

**Review by jflann — CHANGES_REQUESTED** [review]

**Review by jflann — APPROVED** [review]

**Review by jflann — APPROVED** [review]

**Pull request merged**